landfill.py

```diff
@@ -273,9 +273,10 @@ def get_tables(self, models):
         '''
         Extract all peewee models from the passed in module
         '''
-        return { obj._meta.db_table : obj for obj in
-                models.__dict__.itervalues() if
-                isinstance(obj, peewee.BaseModel) and
+        return { obj._meta.table_name : obj for obj in
+                models.__dict__.values() if
+                hasattr(obj, '_meta') and
```

Comment:
In my app, I was getting an instance of `BaseModel` called `Model` which had no `_.meta`